test/field/detail/CMSParameterizedField.i.hh

```diff
@@ -0,0 +1,118 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file CMSParameterizedField.cc
+//---------------------------------------------------------------------------//
+#include "CMSParameterizedField.hh"
+
+#include "base/Units.hh"
+#include <math.h>
```

Comment:
Use `cmath` instead of `math.h` to keep everything in the `std` namespace.

Reply:
Done.